scripts/jenkins/jenkinsfiles/Jenkinsfile-Release

```diff
@@ -0,0 +1,369 @@
+def NODE_LABEL = 'docker'
+// FIXME
```

Comment:
What does need to be fixed?

Reply:
Target S3 bucket, now it points to test.0xdata.com

Reply:
Can this be a parameter? We also need to think about testing the release script (without modifications if possible).

Reply:
I've added TEST_RELEASE parameter. If set then:
  * skip upload to PyPI and Conda, even if UPLOAD_PYPI/UPLOAD_CONDA is set, just build the packages
  * upload to test bucket instead of regular one.

Is this OK @michalkurka?